slackify-html.js

```diff
@@ -147,9 +146,25 @@ function walk(dom, nesting) {
             }
             break;
           case 'h1':
+            content = walk(el.children);
+            content = content.replace(/\*/g, '');
+            out += '*' + content + '*\n';
+            break;
           case 'h2':
+            content = walk(el.children);
+            content = content.replace(/\*/g, '')
+            out += '*' + content + '*\n';
+            break;
           case 'h3':
+            content = walk(el.children);
+            content = content.replace(/\*/g, '')
+            out += '*' + content + '*\n';
+            break;
           case 'h4':
+            content = walk(el.children);
+            content = content.replace(/\*/g, '')
+            out += '*' + content + '*\n';
+            break;
```

Comment:
doesn't this discard all the addional logic below for handling various whitespace in the nested content?

Reply:
@jahlbornG hm good point. i'm gonna move it back and attempt to add a couple more tests to see what happens to the whitespace

Reply:
yep, added some tests and was able to verify that the whitespace can get messed up if i do it this way.